src/ompl/tools/thunder/src/Thunder.cpp

```diff
@@ -214,8 +232,8 @@ ompl::base::PlannerStatus ompl::tools::Thunder::solve(const base::PlannerTermina
     {
         OMPL_DEBUG("Thunder: stopping only after all threads report a solution");
     }
-
-    lastStatus_ = pp_->solve(ptc, hybridize_);  //, hybridize);
+    
+    lastStatus_ = pp_->solve(ptc, minSolCount_, maxSolCount_, hybridize_);  //, hybridize);
```

Comment:
is the `hybridize_` setting documented anywhere?

Reply:
// There are two modes for running parallel plan - one in which both threads are run until they both return a result
    // and/or fail
    // The second mode stops with the first solution found - we want this one
    if (!hybridize_)
    {
        // If \e hybridize is false, when the first solution is found, the rest of the planners are stopped as well.
        OMPL_DEBUG("Thunder: stopping when first solution is found from either thread");
    }
    else
    {
        OMPL_DEBUG("Thunder: stopping only after all threads report a solution");
    }

Reply:
in ParallelPlan.h it's documented